src/everest/detached/__init__.py

```diff
@@ -54,35 +52,7 @@ async def start_server(config: EverestConfig, debug: bool = False) -> Driver:
     """
     Start an Everest server running the optimization defined in the config
     """
-    if server_is_running(
-        *ServerConfig.get_server_context(config.output_dir)
-    ):  # better safe than sorry
-        return
-
-    log_dir = config.log_dir
-
-    configure_logger(
-        name="res",
-        file_path=os.path.join(log_dir, "everest_server.log"),
-        log_level=logging.INFO,
-        log_to_azure=True,
-    )
-
-    configure_logger(
-        name=__name__,
-        file_path=os.path.join(log_dir, "simulations.log"),
-        log_level=logging.INFO,
-    )
-
```

Comment:
Are these logs now in everserver? or is everest_server.log removed

Reply:
Nothing was logging to any of these logging endpoints so these files were always empty so I just removed them.